geopyspark/geotrellis/render.py

```diff
@@ -49,7 +54,7 @@ def get_hex(geopysc, ramp_name, num_colors=None):
         return list(geopysc._jvm.geopyspark.geotrellis.ColorRamp.getHex(ramp_name))
 
 
-class PngRDD(object):
+class PngRDD(RDDWrapper):
```

Comment:
I'm not sure if `PngRDD` is really a duck in this case. The `PngRDD` classes on the Scala side don't inherit or interact from/with the `TileRDD` classes. In addition, `PngRDD` has different initialization and inner workings than either `RasterRDD` or `TiledRasterRDD`.

What is your opinion on this @echeipesh ?